invenio_records_rest/views.py

```diff
@@ -74,6 +76,12 @@ def validation_error(error):
         """Catch validation errors."""
         return RESTValidationError().get_response()
 
+    @blueprint.errorhandler(RequestError)
+    def elasticsearch_badrequest_error(error):
+        """Catche problems in ElasticSearch queries."""
+        description = gettext(error.error)
```

Comment:
Can you provide some examples of what the text is. This error message is for end-users so it's very important that it's something as simple as "The syntax of the search query is invalid."